### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by poojakelkar**
signup

### CODE REVIEW

File: src/pages/Register/index.jsx [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+  AlreadyAc,
+  Button,
+  Checkbox,
+  Container,
+  Element,
+  Form,
+  Heading,
+  Input,
+  Wrapper,
+} from "./styles";
```
Comment (ivinayakg) [inline src/pages/Register/index.jsx:14]: this is really nice that you have created styled-components

File: src/pages/Register/styles.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import styled from "styled-components";
+import { mobile } from "../../responsive";
```
Comment (ivinayakg) [inline src/pages/Register/styles.jsx:2]: i didnt knew you could do this

**Review by ivinayakg — APPROVED** [review]

File: src/pages/Register/index.jsx (hunk 1 above)
Comment (poojakelkar) [reply src/pages/Register/index.jsx:14]: Thanks @ivinayakg

File: src/pages/Register/styles.jsx (hunk 2 above)
Comment (poojakelkar) [reply src/pages/Register/styles.jsx:2]: yeah, I want to create one project with a styled component so I tried this one.

**Review by Rishi05051997 — APPROVED** [review]
Looks Good, clean code & Styling

**Comment by poojakelkar** [comment]
thank @Rishi05051997

**Pull request merged**